src/ResourceManager/Compute/Commands.Compute/Generated/ContainerService/Config/RemoveAzureRmContainerServiceAgentPoolProfileCommand.cs

```diff
@@ -42,6 +42,7 @@ public class RemoveAzureRmContainerServiceAgentPoolProfileCommand : Microsoft.Az
         [Parameter(
             Mandatory = true,
             Position = 1,
+            ParameterSetName = "NameParameterSet",
```

Comment:
We generated these cmdlets.  For Removing there are can be more than one way to removing (for example, removing with a name and removing with a id), so we added parameter set name for each removing way.